### PR TITLE
Added issue counter to JS and SCSS builds

### PR DESCRIPTION
**Pull request by keichinger**
Added a simple way to print the total amount of SCSS and JS issues that have been detected by JsHint and SCSS-Lint.

Unfortunately I couldn't manage to print both statistics at the very end of both operations as they're both asynchronous. However, they'll be printed at the end of each single operation so it is still useful.

### CODE REVIEW

File: tasks/scss.js [hunk 1]
```diff
@@ -17,12 +17,66 @@ var cssMin = require("gulp-minify-css");
...
+    {
+        outputColor = gulpUtil.colors.red;
+    }
+
+    gulpUtil.log(gulpUtil.colors.yellow('»»'), 'Total CSS issues:', outputColor(totalIssues));
+
+    // Reset the issue count so we don't increment it every time a file has been modified while it is being watched
+    totalIssues = 0;
+
+    return through.obj();
```
Comment (apfelbox) [inline tasks/scss.js]: Why is this needed?

Comment (keichinger) [inline tasks/scss.js]: Weirdly this function wasn't executed for some reason when I either tried to .pipe() it or execute it via .on('end', function () { ... });

Maybe it's a simple thing I missed in the whole gulp concept of how to synchronously pipe things and I can easily omit it?

Comment (apfelbox) [inline tasks/scss.js]: Well, you never pass this return value anywhere, so... ?

**Comment by keichinger** [comment]
I've updated the PR to no longer rely on through2. I have no idea why it wasn't working before but now, without through2, it is working as intended.

File: tasks/scss.js [hunk 2]
```diff
@@ -78,12 +129,16 @@ function lintFiles (src)
...
         .pipe(scssLint({
-            config: __dirname + "/../config/scss-lint.yml"
-        }));
+            config: __dirname + "/../config/scss-lint.yml",
+            customReport: issueCountReporter
+        }))
+        .on('end', function ()
+        {
+            reportTotalIssueCount();
+        });
```
Comment (apfelbox) [inline tasks/scss.js]: You should write this as

``` js
.on('end', reportTotalIssueCount);
```
